app/upgrades/v28/upgrades.go

```diff
@@ -0,0 +1,70 @@
+package v28
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	stakeibckeeper "github.com/Stride-Labs/stride/v27/x/stakeibc/keeper"
+)
+
+var (
+	UpgradeName = "v28"
+
+	OsmosisChainId = "osmosis-1"
+
+	// Redemption rate bounds updated to give ~3 months of slack on outer bounds
+	RedemptionRateOuterMinAdjustment = sdk.MustNewDecFromStr("0.05")
+	RedemptionRateOuterMaxAdjustment = sdk.MustNewDecFromStr("0.25")
+
+	// Osmosis will have a slighly larger buffer with the redemption rate
+	// since their yield is less predictable
+	OsmosisRedemptionRateBuffer = sdk.MustNewDecFromStr("0.02")
```

Comment:
Do we need this? Outer bounds are already wide

Reply:
I don't think so, we've just separated out Osmosis in the past. I'll remove it from this one for simplicity